runtime/src/lib.rs

```diff
@@ -898,3 +918,27 @@ cumulus_pallet_parachain_system::register_validate_block! {
     Runtime = Runtime,
     BlockExecutor = cumulus_pallet_aura_ext::BlockExecutor::<Runtime, Executive>,
 }
+
+// tests
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    // RUNTIME_API_VERSIONS constant is generated by a macro and is private.
```

Comment:
this is the only test in `src/lib.rs` for runtime, the reason is explained in the comment.
I couldn't find a good way to move this to integration tests due to the visibility of the variable.

I think it's better to include it in here than to remove and reduce coverage.
If you guys think it's ok, let's leave it as is.